app/javascript/lib/sentry-logging.js

```diff
@@ -114,8 +114,8 @@ export function setupSentry() {
   Sentry.init({
     dsn: 'https://a713dcf8bbce4a26aa1fe3bf19008d26@o54426.ingest.sentry.io/1424198',
     integrations: [
-      new Sentry.BrowserTracing(),
```

Comment:
I assume keeping [browser tracing](https://docs.sentry.io/platforms/javascript/tracing/) needs more than merely updating `@sentry/tracing` to the same version as `@sentry/react`.  If so, this removal makes sense to me, because I can't think of any cases where we've gotten high-priority insight from the granular performance metrics it provides.

Reply:
`@sentry/tracing` has been deprecated since `7.47`, and while upgrading I received several warnings to that effect.  Removing it seems to have no apparent consequences.

https://github.com/getsentry/sentry-javascript/blob/7.47.0/MIGRATION.md/#remove-requirement-for-sentrytracing-package-since-7460